server/app/services/languageProcessor/LUIS.java

```diff
@@ -0,0 +1,102 @@
+package services.languageProcessor;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import play.Configuration;
+import play.api.Play;
+import play.libs.Json;
+import play.libs.ws.WSClient;
+import play.libs.ws.WSRequest;
+import play.libs.ws.WSResponse;
+import play.mvc.Result;
+import services.IntentEntity;
+import services.queryHandler.JiraApiFetchInfo;
+import java.util.concurrent.CompletionStage;
+
+import static play.mvc.Results.ok;
+
+/**
+ * Created by sabinapokhrel on 11/21/16.
+ */
+public class LUIS {
+
+  private Configuration configuration = Play.current().injector().instanceOf(Configuration.class);
```

Comment:
Please provide this in the constructor so that no manual injection is happening within this class. Even better would be to define a `LuisConfiguration` class to own the responsibility of processing the configuration.